### DISCUSSION TITLE
Startup errors

### DISCUSSION THREAD

**Discussion by zhouyuansheng0618** [StackStorm/st2#5543] [Packs]
pymongo.errors.OperationFailure: no such command: 'listindexes', full error: SON([('ok', 0.0), ('errmsg', "no such command: 'listindexes'"), ('code', 59), ('codeName', 'CommandNotFound')])

**Comment by arm4b** [comment]
Could you provide more info about the environment, deployment method, versions used, and conditions that resulted in this error message?
